Delicate-Butterflies/Bangazon-Workforce
# Sb generating seeders

Related Issue: Seeding data

Description: Created Seeder files. If a model is not yes available then ```db:seed:all``` command WILL FAIL. This can be resolved by running ```db:seed --seed filenameOfSeeder``` for available models. 

Chai/Mocha Tests File: N/A

PR Testing Steps:
- ```DB:MIGRATE:UNDO:ALL``` BEFORE FETCHING THIS BRANCH DOWN OR IT WILL BREAK YOUR DATABASE.
- Fetch the branch
- run ```db:migrate```
- run ```db:seed:all`` the seed will FAIL on the first unavailable model, but will SUCCEED for all before that
- run ```psql``` and connect to the database
- run ```select * from tablename``` where tablename is the table you wish to inspect. If it is populated seeding succeeded! If not it failed.
- Inform me if I need comments. Creating seeders doesn't seem to require them.

Check list before you make a PR:
